src/katgpucbf/utils.py

```diff
@@ -155,7 +155,7 @@ def comma_split(
         times.
     """
 
-    def func(value: str) -> list[_T]:  # noqa: D102
+    def func(value: str) -> list[_T]:
```

Comment:
Why does this function lose the `D102` marker entirely, but `update_aggregate` (L191) retains it?

Reply:
Ah, looking at it again, and re-reading https://www.pydocstyle.org/en/2.1.1/error_codes.html
* Perhaps it's because `func` is not a public method?

Reply:
Yes, I think it was a bug in pydocstyle that it required this. Or possibly someone pro-actively added the noqa when it wasn't needed.

One of ruff's optional warnings is to complain about an unneeded noqa. I turned that on to identify where I could delete. I didn't commit it, because it complained about all the noqa's in the test files where we told it not to check, but my intention is to enable it after NGC-1465 is addressed.